app/services/analytics.rb

```diff
@@ -21,6 +21,7 @@ def track_event(event, attributes = {})
       event_properties: attributes.except(:user_id).compact,
       new_event: first_event_this_session?,
       path: request&.path,
+      service_provider: sp,
```

Comment:
I had to triple check, but I forgot what the type of `sp` was as an attribute, since we're here, WDYT of adding type documentation to the initialzer for this class like
```
# @param sp [String] issuer string for the service provider for this analytics event
```

Reply:
On that note, would it be worthwhile to rename the `sp` _param_ to `sp_issuer` or just `issuer`?